Jenkinsfile.integration

```diff
@@ -50,16 +36,16 @@ node('lisk-integration') {
 			try {
 				deleteDir()
 				checkout scm
-
-				sh '''
-					# Cycle postgres, install deps and prepare config
-					sudo service postgresql restart
-					rsync -axl -e "ssh -oUser=jenkins" master-01:/var/lib/jenkins/lisk/node_modules/ "$WORKSPACE/node_modules/" || true
-					npm install
-					cp test/data/config.json test/data/genesis_block.json .
-				'''
+				// Cycle postgres, restore chache install deps and prepare config
+				sh 'sudo service postgresql restart'
+				cache_file = restoreCache("config.json")
```

Comment:
With recent changes into core there will be no `config.json` on root directory. Does that really matters here?

Reply:
Yes I'm aware of that, but this is another issue we need to fix for this branch 1.1.0.  For what matters here I suggest to get it merged and work in the config changes in another PR.